__tests__/app.test.js

```diff
@@ -44,4 +43,44 @@ describe("GET /api/topics", () => {
         })
       })
   })
+})
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an article object each with the correct properties", () => {
+    return request(app)
+      .get('/api/articles/1')  
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject
+        ({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
Considering the request we are making here - do we want the article id to be ANY number? Or a specific one? because if article 5 is returned for example, this test would pass but we wouldn't expect it to. 
Keeping the other keys flexible is fine though as we aren't concerned with what the specific title or author etc is - these can change in the test data and it's not a problem but if we request article 1 then we should always get back article number 1